dialogue-clients/src/main/java/com/palantir/dialogue/clients/DialogueDnsResolutionWorker.java

```diff
@@ -104,5 +111,7 @@ private synchronized void doUpdate(@Nullable INPUT updatedInputState) {
                 log.info("Attempted to update DNS output refreshable which has already been garbage collected");
             }
         }
+        long end = System.nanoTime();
+        updateTimer.update(Duration.ofNanos(end - start));
```

Comment:
We may want to put this inside the `if (inputState != null) {` conditional to make sure we're actually measuring work